### PR TITLE
Updated from source

### PR DESCRIPTION
**Pull request by makhotkin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pathofexile-dat-schema",
-  "version": "6.0.1",
+  "version": "6.0.2",
```
Comment (SnosMe) [inline package.json:3]: Don't need to update this one, schema definitions are separate from package.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pathofexile-dat-schema",
-  "version": "7.0.0",
+  "version": "7.0.1",
```
[the remark this answers is not in the document]
Comment (makhotkin) [reply package.json:3]: Will fix, I read your msg after resolving the conflict

[the remark this answers is not in the document]
Comment (makhotkin) [reply package.json]: check the diff now

**Comment by makhotkin** [comment]
Looks like the PR is complete now

**Pull request merged**